### PR TITLE
Added change detection bin script 

### PR DESCRIPTION
**Pull request by Kelvinrr**

### CODE REVIEW

File: bin/acn_cd.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        exit()
+
+    before_proj_geo = GeoDataset(before_proj)
+    after_proj_geo = GeoDataset(after_proj)
+    ret = _cd_functions_[args.method](before_proj_geo, after_proj_geo, **config[args.method])
+
+    # for now, write out raster files assuming okb
+    # make it match one of the projected images
+    match_srs = before_proj_geo.dataset.GetProjection()
+    match_gt = before_proj_geo.geotransform
```
Comment (Kelvinrr) [inline bin/acn_cd.py:104]: gonna update this after homogenizing the returns. Something to talk about tomorrow AM.

File: autocnet/cg/change_detection.py [hunk 2]
```diff
@@ -92,6 +101,12 @@ def okubogar_detector(image1, image2, nbins=50, extractor_method="orb", image_fu
      if isinstance(image2, GeoDataset):
          image2 = image2.read_array()
 
+     if isinstance(image_func, str):
+         try:
+             image_func = func_map[image_func]
+         except Exception as e:
```
Comment (AustinSanders) [inline autocnet/cg/change_detection.py:107]: Are we expecting anything other than KeyError here?

File: bin/acn_cd.py [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+from autocnet.utils.utils import bytescale
+
+import numpy as np
+
+import pysis
+from pysis.exceptions import ProcessError
+from pysis import isis
+
+import warnings
+warnings.simplefilter("ignore")
```
Comment (AustinSanders) [inline bin/acn_cd.py:28]: Which warnings are we squelching?

Comment (Kelvinrr) [reply bin/acn_cd.py:28]: Pandas warnings about df copies when Autocnet is generating matches. They get obnoxious after a while and they are not describing real problems.

File: autocnet/cg/change_detection.py (hunk 2 above)
Comment (Kelvinrr) [reply autocnet/cg/change_detection.py:107]: no, I can make it more specific, but I don't think it matters in this case.

**Pull request merged**